blocks/init/src/blocks/components/video-iframe/components/video-iframe-editor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import classnames from 'classnames';
+
+export const VideoIframeEditor = (props) => {
+  const {
+    id,
+    url,
+    aspectRatio,
+    blockClass,
+    title,
+  } = props;
+
+  const componentClass = 'video-iframe';
+
+  const videoClass = classnames(
+    componentClass,
+    `${componentClass}__video-ratio--${aspectRatio}`,
+    `${blockClass}__${componentClass}`,
+  );
+
+  return (
+    <div className={videoClass}>
+      <iframe
+        className={`${componentClass}__iframe`}
+        src={`${url}${id}`}
+        title={title}
+        frameBorder="0"
+        allow="autoplay; fullscreen"
+        allowFullScreen
```

Comment:
Should these be properties as well?

Reply:
This is just in the editor I believe, on frontend you be oembed's output.

Reply:
I was thinking about the editor, like if you'd like to change the autoplay option etc.